requirements-test.txt

```diff
@@ -15,4 +15,4 @@ pytest == 5.4.1
 pytest-cov == 2.10.0
 pytest-mpi == 0.4
 pytest-timeout == 1.4.2
-timm >= 0.4.5
+git+git://github.com/rwightman/pytorch-image-models@v0.4.5#egg=timm
```

Comment:
The PyPi `timm` package pins to a specific PyTorch / torchvision versions, so I'm installing it from github instead